### DISCUSSION TITLE
【主机】编辑文件保存失败时仍然提示成功

### DISCUSSION THREAD

**Discussion by Pandaft** [1Panel-dev/1Panel#6507] [Q&A]
### 1Panel 版本

v1.10.15-lts

### 问题描述

在保存编辑的文件时，即使响应是错误的（比如 502 Bad Gateway），也会提示“更新成功”，并且会路由跳转到首页概览。
实际保存失败，并且之前所做的变更将会丢失。

### 重现步骤

1. 打开任意文件进行编辑
2. 使 1Panel 的  /api/v1/files/save 接口返回 502 错误（其他错误未测试），参考方式：
    - 更改网络状态，比如在我这是更换代理节点
    - 使用 Nginx 等将上述接口路径的请求返回 502 错误
3. 保存编辑

### 期待的正确结果

提示具体错误，比如 “502 网关错误” 等，并且不关闭编辑窗口，避免丢失所做的变更。

### 附加信息

![图片](https://github.com/user-attachments/assets/f4050423-101c-464b-8598-2c8f55473fd2)

**Comment by wanghe-fit2cloud** [comment]
1Panel 是通过 IP 还是反代域名访问？
> 可以提供一个录屏或者动图
文件 `编辑` - `保存` 操作不会关闭编辑窗口。

**Reply by Pandaft** [reply]
通过 IP:PORT 访问，详见录屏：

https://github.com/user-attachments/assets/c2a0bf1e-f4a0-48c4-a5bb-41980bed35fc
